app/models/user.rb

```diff
@@ -0,0 +1,5 @@
+class User < ApplicationRecord
+  validates :username, presence: true
+  validates :password, presence: true
+  validates :email, presence: true
+end
```

Comment:
OPTIONAL
- Always keep your code DRY. You can simply write the three lines like this
`validates: :username, :password, :email, presence: true`